common/benchmark_utils.py

```diff
@@ -53,16 +56,34 @@ def get_builder_image_url(benchmark, fuzzer, docker_registry):
         docker_registry=docker_registry, fuzzer=fuzzer, benchmark=benchmark)
 
 
-def validate(benchmark):
-    """Return True if |benchmark| is a valid fuzzbench fuzzer."""
+def validate_name(benchmark):
+    """Returns True if |benchmark| is a valid fuzzbench benchmark name."""
     if VALID_BENCHMARK_REGEX.match(benchmark) is None:
         logs.error('%s does not conform to %s pattern.', benchmark,
                    VALID_BENCHMARK_REGEX.pattern)
         return False
-    if benchmark in get_all_benchmarks():
-        return True
-    logs.error('%s must have a benchmark.yaml.', benchmark)
-    return False
+    return True
+
+
+def validate(benchmark):
+    """Returns True if |benchmark| is a valid fuzzbench benchmark."""
+    if not validate_name(benchmark):
+        return False
+    if benchmark not in get_all_benchmarks():
+        logs.error('%s must have a benchmark.yaml.', benchmark)
+        return False
+    try:
+        get_fuzz_target(benchmark)
```

Comment:
maybe add get_project as well.

Reply:
I don't think project is actually mandatory. The function is actually dead code. I'm removing this function.